linter.py

```diff
@@ -120,6 +138,16 @@ def ensure_plugin_installed(self) -> bool:
         self.notify_unassign()  # Abort linting without popping error dialog
         raise PermanentError()
 
+    def get_stdin_filename(self) -> str | None:
```

Comment:
`str | None` is the newest Python syntax. We run 3.3 (or in some future 3.8).  Doesn't that throw? Did you restart Sublime?  To what I know, for Python 3.3 it must even be a type comment, like

```
def lirum():
	# type: () -> Optional[str]
```